src/cljs/org/broadinstitute/firecloud_ui/page/import_data.cljs

```diff
@@ -0,0 +1,36 @@
+(ns org.broadinstitute.firecloud-ui.page.import-data
+  (:require
+    [clojure.string]
+    [dmohs.react :as react]
+    [org.broadinstitute.firecloud-ui.common :as common]
+    [org.broadinstitute.firecloud-ui.common.components :as comps]
+    [org.broadinstitute.firecloud-ui.common.style :as style]
+    [org.broadinstitute.firecloud-ui.common.table :as table]
```

Comment:
We don't have optimize imports. Sorry. No need to clean these up, but do so if you feel like it.